### PR TITLE
New version: Bambulab.Bambustudio version 1.8.4

### PR DESCRIPTION
**Pull request by mbilal234**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138519)

### CODE REVIEW

**Comment by mbilal234** [comment]
image

**Comment by stephengillie** [comment]
Hi @mbilal234,

> This PR's version number 1.8.4 has 3 parameters (sets of numbers between dots - major, minor, etc), which is less than the current manifest's version 1.7.7.89, which has 4 parameters.

Does this PR's version number **exactly** match the version reported in the Apps & features Settings page? (Feel free to attach a screenshot.)

(Automated response - build 726)

**Comment by stephengillie** [comment]
> image

Thanks for supplying a screenshot of the repo. Better would be a screenshot of the Apps & features Settings page. We need to verify that what's written to Registry matches the manifest contents - matching here drives upgrades, and incorrect information here can cause the "upgrade always available" error among other problems.

**Comment by mbilal234** [comment]
image


The installer lists it as 01.08.04.51

Should I change the yaml files to reflect the same?

**Comment by stephengillie** [comment]
Yes, as the installer matches the Apps & features Settings page. 
image

File: manifests/b/Bambulab/Bambustudio/1.8.4/Bambulab.Bambustudio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created with YamlCreate.ps1 v2.3.3 $debug=QUSU.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.6.0/manifest.installer.1.6.0.json
+
+PackageIdentifier: Bambulab.Bambustudio
+PackageVersion: 1.8.4
```
Comment (stephengillie) [inline manifests/b/Bambulab/Bambustudio/1.8.4/Bambulab.Bambustudio.installer.yaml:5]:
```suggestion
PackageVersion: 01.08.04.51
```
Please update file name and path to match this change.

File: manifests/b/Bambulab/Bambustudio/1.8.4/Bambulab.Bambustudio.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.3.3 $debug=QUSU.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.6.0/manifest.defaultLocale.1.6.0.json
+
+PackageIdentifier: Bambulab.Bambustudio
+PackageVersion: 1.8.4
```
Comment (stephengillie) [inline manifests/b/Bambulab/Bambustudio/1.8.4/Bambulab.Bambustudio.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 01.08.04.51
```
Please update file name and path to match this change.

File: manifests/b/Bambulab/Bambustudio/1.8.4/Bambulab.Bambustudio.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.3.3 $debug=QUSU.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.6.0/manifest.version.1.6.0.json
+
+PackageIdentifier: Bambulab.Bambustudio
+PackageVersion: 1.8.4
```
Comment (stephengillie) [inline manifests/b/Bambulab/Bambustudio/1.8.4/Bambulab.Bambustudio.yaml:5]:
```suggestion
PackageVersion: 01.08.04.51
```
Please update file path to match this change.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-10T06:41:17.8864198Z ##[error] One or more errors occurred. (Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\b\Bambulab\Bambustudio\1.8.4', expected 'manifests\b\Bambulab\Bambustudio\01.08.04.51') Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\b\Bambulab\Bambustudio\1.8.4', expected 'manifests\b\Bambulab\Bambustudio\01.08.04.51'

(Automated response - build 729.)

**Comment by stephengillie** [comment]
Hi @mbilal234,

The file path still needs to be changed. 

Path: `/1.8.4` 
Version: `01.08.04.51` 

These need to match. This can be achieved through either creating a new folder with the correct name and moving the files there, or through renaming the current folder.

**Comment by mbilal234** [comment]
> Hi @mbilal234,
> 
> The file path still needs to be changed.
> 
> Path: `1.8.4` Version: `01.08.04.51`
> 
> These need to match. This can be achieved through either creating a new folder with the correct name and moving the files there, or through renaming the current folder.

Ok, I shall do that

**Review by stephengillie — APPROVED** [review]

**Pull request merged**